test_assets/PrimvarInterpolation/README.md

```diff
@@ -0,0 +1,221 @@
+
+```
+int[] faceVertexCounts = [4, 4]
+# int[] faceVertexCounts = [A, B]
+
+color3f[] primvars:displayColor = [(1, 0.3, 0.3), (0.3, 0.3, 1)] (
+    interpolation = "uniform"
+)
+#color3f[] primvars:displayColor = [A, B] (
+#    interpolation = "uniform"
+#)
+```
+
+## varying
+
+![varying interpolation](images/varying.png)
+
+```
+color3f[] primvars:displayColor = [(1, 0, 0), (1, 1, 0), (1, 1, 1), (0, 1, 0), (1, 0, 1), (0, 0, 1)]  (
+    interpolation = "varying"
+)
+```
+
+**Pixar's definition**
+> Four values are interpolated over each uv patch segment of the surface. Bilinear interpolation is used for interpolation between the four values.
+
+**Size of primvar array**
+
+number of vertices on the mesh, which is the same as the length of **points** array
+
+**Notes**
+
+With varying interpolation, one value is specified for each vertex of the mesh. The order of the array matches the order of the **points** array:
+
+```
+point3f[] points = [(-1, 0, 0), (-1, 1, 0), (0, 1, 0), (0, 0, 0), (1, 1, 0), (1, 0, 0)]
+# point3f[] points = [N, M, O, P, Q, R]
+
+color3f[] primvars:displayColor = [(1, 0, 0), (1, 1, 0), (1, 1, 1), (0, 1, 0), (1, 0, 1), (0, 0, 1)]  (
+    interpolation = "varying"
+)
+#color3f[] primvars:displayColor = [N, M, O, P, Q, R]  (
+#    interpolation = "varying"
+#)
+```
+
+## vertex
+
+![vertex interpolation](images/vertex.png)
+
+```
+color3f[] primvars:displayColor = [(1, 0, 0), (1, 1, 0), (1, 1, 1), (0, 1, 0), (1, 0, 1), (0, 0, 1)] (
+    interpolation = "vertex"
+)
+```
+
+**Pixar's definition**
+> Values are interpolated between each vertex in the surface primitive. The basis function of the surface is used for interpolation between vertices.
+
+**Size of primvar array**
+
+number of vertices on the mesh, which is the same as the length of **points** array
+
+**Notes**
+
+With vertex interpolation, one value is specified for each vertex of the mesh. The order of the array matches the order of the **points** array:
+
+```
+point3f[] points = [(-1, 0, 0), (-1, 1, 0), (0, 1, 0), (0, 0, 0), (1, 1, 0), (1, 0, 0)]
+# point3f[] points = [N, M, O, P, Q, R]
+
+color3f[] primvars:displayColor = [(1, 0, 0), (1, 1, 0), (1, 1, 1), (0, 1, 0), (1, 0, 1), (0, 0, 1)]  (
+    interpolation = "vertex"
+)
+#color3f[] primvars:displayColor = [N, M, O, P, Q, R]  (
+#    interpolation = "vertex"
+#)
+```
+
+You probably noticed that **varying** and **vertex** interpolation are quite similar. The differences between them are subtle, and relate to how they interact with subdivision on the mesh. You can see some differences in the example below, where the Mesh prims have a duplicate to their side that sets `token interpolateBoundary = "edgeOnly"`, which causes the corners of the mesh to be rounded when subdivision is applied. **varying** is on top, **vertex** is on the bottom. Note the stronger colors in the "corners" of the rounded mesh with **varying** interpolation.
```

Comment:
I think it's important to call out here that for DCC tools exporting data, if the exported data is expected to be a polygon mesh (and the surface is explicitly not intended to be subdivided), then primvar interpolation should be set to "varying" and the mesh subdivision scheme should be set to "none", to ensure the mesh renders as intended when downstream consumers DO support subdivision.

As I'm writing this, I'm realizing there's a whole discussion here about how to handle data for export if it originates from the tool or if it came from an existing USD file, and might be out of scope for this...